ci/autoscaler/scripts/deploy-autoscaler.sh

```diff
@@ -20,6 +20,7 @@ ops_files=${OPS_FILES:-"${autoscaler_dir}/operations/add-releases.yml\
   ${autoscaler_dir}/operations/remove-metricsserver.yml\
   ${autoscaler_dir}/operations/remove-metricsgateway.yml\
   ${autoscaler_dir}/operations/enable-log-cache-via-uaa.yml\
+  ${autoscaler_dir}/operations/enable-metricsforwarder-syslog.yml\
```

Comment:
This causes the acceptance tests to run exclusively with the setup of sending the custom metrics to the syslog-server.

@silvestre do we still need to test the setup in our acceptance tests where the metricsforwarder is configured to send metrics to the loggregator metron agent?

Reply:
Please also adapt the [Concourse pipeline](https://github.com/cloudfoundry/app-autoscaler-release/blob/metricforwarder-on-cf/ci/autoscaler/pipeline.yml#L15-L27) as well by adding a new `acceptance-log-cache-syslog` job testing the new feature and maybe rename the existing `acceptance-log-cache` to `acceptance-log-cache-metron`.